godotproject/Star.gd

```diff
@@ -0,0 +1,6 @@
+extends Area2D
```

Comment:
Hey can you break the Star into its own scene? If you go into Level0 in the editor, there should be an option when right-clicking the star's root node called "Save Branch To Scene". You might need to reset the Node2D -> Transform property in the newly created scene to 0.

Reply:
Done! Nice catch